From 9a47207482e51ee9cbf55b76354ce7e7b798e077 Mon Sep 17 00:00:00 2001
From: Octavian Purdila <octavian.purdila@nxp.com>
Date: Tue, 7 Feb 2017 17:53:20 +0200
Subject: [PATCH 0041/5242] MLK-13863: ARM: imx6ul: reserve ocram space for
 low power mode

commit  600466b022a8f669e346c1026306e596aa89bff2 from
https://source.codeaurora.org/external/imx/linux-imx.git

This patches adds two new device nodes and carves out space used by
the low power mode code.

Signed-off-by: Octavian Purdila <octavian.purdila@nxp.com>

During 4.14 rebase moved this patch earlier because otherwise 6ul
suspend code breaks at boot.

Signed-off-by: Leonard Crestez <leonard.crestez@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6ul.dtsi |   14 ++++++++++++--
 1 file changed, 12 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx6ul.dtsi b/arch/arm/boot/dts/imx6ul.dtsi
index 2a299bb..a9dce60 100644
--- a/arch/arm/boot/dts/imx6ul.dtsi
+++ b/arch/arm/boot/dts/imx6ul.dtsi
@@ -183,9 +183,19 @@
 			fsl,max_ddr_freq = <400000000>;
 		};
 
-		ocram: sram@900000 {
+		ocrams: sram@900000 {
+			compatible = "fsl,lpm-sram";
+			reg = <0x00900000 0x4000>;
+		};
+
+		ocrams_ddr: sram@904000 {
+			compatible = "fsl,ddr-lpm-sram";
+			reg = <0x00904000 0x1000>;
+		};
+
+		ocram: sram@905000 {
 			compatible = "mmio-sram";
-			reg = <0x00900000 0x20000>;
+			reg = <0x00905000 0x1B000>;
 		};
 
 		dma_apbh: dma-apbh@1804000 {
-- 
1.7.9.5

